src/TokenManagers/MinecraftBedrockTokenManager.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs')
+const path = require('path')
+const debug = require('debug')('xboxlive-auth')
+const fetch = require('node-fetch')
+
+const { Authentication } = require('../common/Constants')
+const { checkStatus } = require('../common/Util')
+
+class BedrockTokenManager {
+  constructor (clientPublicKey, cacheLocation) {
```

Comment:
Remove first parameter here from the constructor, it’s not used

Reply:
Are you sure? It looks used to me